composer.json

```
@@ -52,7 +52,7 @@
     "doctrine/annotations": "^1.6.0",
     "doctrine/collections": "^1.5.0",
     "doctrine/persistence": "^2.1.0",
-    "doctrine/dbal": "^2.11.0",
+    "doctrine/dbal": "^3.1.3",
```

```suggestion
    "doctrine/dbal": "^2.11.0 || ^3.1.3",
```
Any concerns about supporting both?

---

as is that will not work. signatures are to different